src/mam4xx/mo_setext.hpp

```diff
@@ -0,0 +1,142 @@
+#ifndef MAM4XX_MO_SETEXT_HPP
+#define MAM4XX_MO_SETEXT_HPP
+
+#include <haero/math.hpp>
+#include <mam4xx/aero_config.hpp>
+#include <mam4xx/mam4_types.hpp>
+#include <mam4xx/utils.hpp>
+
+namespace mam4 {
+
+namespace mo_setext {
+using View1D = DeviceType::view_1d<Real>;
+using View2D = DeviceType::view_2d<Real>;
+using View3D = DeviceType::view_3d<Real>;
+constexpr int pver = mam4::nlev;
+constexpr int extfrc_cnt = 9;
+constexpr int extcnt = 9; //, & ! number of species with external forcing
```

Comment:
Probably trivial or not worth doing now, but it appears `extcnt` is never used in a meaningful way.